src/other/tables.jl

```diff
@@ -21,28 +21,29 @@ Tables.getcolumn(dfr::DataFrameRow, nm::Symbol) = dfr[nm]
 getvector(x::AbstractVector) = x
 getvector(x) = [x[i] for i = 1:length(x)]
 # note that copycols is ignored in this definition (Tables.CopiedColumns implies copies have already been made)
-fromcolumns(x::Tables.CopiedColumns, names; copycols::Bool=true) =
+fromcolumns(x::Tables.CopiedColumns, names; makeunique::Bool=false, copycols::Bool=true) =
     DataFrame(AbstractVector[getvector(Tables.getcolumn(x, nm)) for nm in names],
```

Comment:
I have noted that this iteration through columns will break if names are not unique. This can be fixed by iterating by index instead.